tests/test_log.py

```diff
@@ -100,3 +105,62 @@ def test_extract_boolean_value(self):
         eis = log.find("EventInfo")
         for ei in eis:
             assert isinstance(ei.time_event_info, bool), "Expected ei.time_event_info to be bool"
+
+class TestNoLogFile:
```

Comment:
Does these tests really work?

Reply:
Yes they do, they run without issues when I run them locally, or is it anything you see that indicates that they should not work?

Reply:
Ok, see comment below.